bedrock/mozorg/redirects.py

```diff
@@ -588,6 +588,10 @@ def decider(request, **kwargs):
     # bug 1003737
     redirect(r'^impressum/?$', '/about/legal/impressum/'),
 
+    # Bug 682619
+    redirect(r'^support/thunderbird/?$', 'https://support.mozilla.org/products/thunderbird'),
```

Comment:
This only catches `support/thunderbird/`, but not things like `support/thunderbird/bugs`.